Runtimes/Core/Readme.md

```diff
@@ -0,0 +1,92 @@
+# SwiftCore
+
+SwiftCore contains core libraries that sit below the platform overlays.
+These include the standard library and associated runtimes, SwiftOnoneSupport,
+and the concurrency runtimes.
+
+## Build Instructions
+
+> [!IMPORTANT]
+> The standard library requires that it is built with a Swift compiler that is
+> at least as new as the standard library sources. You will likely need to
+> build the compiler as you would normally.
+> In these instructions, `<swiftc>` is the path to your just-built Swift
+> compiler.
+
+First create a build directory in the `Core/` directory and navigate to it.
+
+```sh
+mkdir build && cd build
+```
+
+Configure a simple default build. This results in a static build of the standard
+library.
+
+```sh
+cmake -G 'Ninja' -DCMAKE_Swift_COMPILER=<swiftc> ../`
+ninja
+```
+
+To build a dynamic library, pass `-DBUILD_SHARED_LIBS=YES` to CMake.
+
+```sh
+cmake . -DBUILD_SHARED_LIBS=YES
+ninja
```

Comment:
```suggestion
cmake -B build -G Ninja -S . -D CMAKE_Swift_COMPILER=<swiftc> -D BUILD_SHARED_LIBS=YES
ninja -C build
```

This keeps the build rules nearly identical and clearly shows the difference.

Reply:
I don't think I agree. Once you have a build configured, we only need to show what needs to change in order to get the desired effect. The others are just clutter.